modules/sdk/project-templates-freemarker-portlet/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -0,0 +1,2 @@
+.portlet-freemarker {
```

Comment:
The problem is that, without a `src/main/resources/META-INF/resources`, the CSS part of the Maven build fails.